### PR TITLE
Test --help across all commands

### PR DESCRIPTION
**Pull request by dwaynebailey**
This is to tease out any potential config issues in current and future argparse commands.

### CODE REVIEW

**Comment by julen** [comment]
I'm not sure I understand the motivation behind this; mind elaborating?

**Comment by dwaynebailey** [comment]
@julen simple really.  This picked up a number of issues with optparse to argparse migration.  But as a default will run the config setup for any commands we land.  While it doesn't exercise the meat of any command it makes sure we're at least trying to run them and keep them working.

File: tests/commands/minus_minus_help.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+import pytest
+
+from django.core.management import call_command, get_commands
+
+
+@pytest.mark.cmd
+@pytest.mark.parametrize("command,app", [
+    (command, app)
+    for command, app in get_commands().iteritems()
+    if not app.startswith("django.") and not app.startswith("django_")
```
Comment (unho) [inline tests/commands/minus_minus_help.py]: Is there any reason not to use just `not app.startswith("django")`?

File: tests/commands/minus_minus_help.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+])
+def test_initdb_help(capfd, command, app):
+    """Catch any simple command issues"""
+    print "Command: %s, App: %s" % (command, app)
```
Comment (unho) [inline tests/commands/minus_minus_help.py]: Nit, use print as a function.

**Pull request merged**